code/iaas/allocator/pom.xml

```diff
@@ -28,5 +28,10 @@
             <artifactId>cattle-framework-jooq</artifactId>
```

Comment:
I suspect that there might be circular reference dependencies. cattle-iaas-api-logic uses cattle-iaas-logic, and cattle-iaas-logic uses allocator:

```
code/iaas/api-logic/pom.xml:    <artifactId>cattle-iaas-api-logic</artifactId>
code/iaas/logic/pom.xml:            <artifactId>cattle-iaas-allocator</artifactId>
```
Can you please run the check in the eclipse